addons/godot-xr-tools/objects/pickable.gd

```diff
@@ -226,7 +226,8 @@ func pick_up(by: Node3D) -> void:
 	# Skip if not enabled
 	if not enabled:
 		return
-
+	
+	freeze = false
```

Comment:
why?

Reply:
Unfreezes picked up items. You might be wondering where items are frozen. All the spawner items are. Like the patties, buns, drinks, trays, plates...

Reply:
why not use this?
![image](https://github.com/Three-s-A-Crowd-Games/foodtruck-sim/assets/96577754/4b17ead8-ec2f-46e5-baa0-5d597fb05947)

Reply:
Could be used, yes. Dunno.

Reply:
But that is technically more annoying as this then would need to be set manually for trays etc as well.
And that one line shouldn't do any harm AND technically just defaults everyone to unfrozen (which we do) and you COULD freeze if you needed to.
Don't know what's better. Having that one line or editing all other frozen objects (tray, friesboxes, food (only food.tscn), plates)

Reply:
okay idc. Hope it won't bite us in the ass